packages/cozy-client/src/devtools/Queries.jsx

```diff
@@ -43,6 +43,15 @@ const TableCell = withStyles({
   }
 })(MuiTableCell)
 
+const getClassNameForExecutedTimesQuery = time => {
+  if (time <= 100) {
+    return 'u-valid u-success'
+  } else if (time > 100 && time < 500) {
+    return 'u-warn u-warning'
+  } else {
+    return 'u-danger u-error'
+  }
```

Comment:
500 seems already a bit high :D This execution time is not counting the network time etc.. It is only inside couchdb right ?

Reply:
Yep this is only the time inside couchdb. OK to move it down to 250!

Reply:
Yes it only includes couchdb time. It is high indeed, but you might have legitimate queries taking some time, for instance because you requested all the docs at once. Typically, I wouldn't be surprised that the query getting all the banks operations might take hundreds of ms for 10k+ docs :)

Reply:
I think we should have a warning for the query in Banks, we know this is a problem to retrieve all those docs.

Reply:
Yes a warning looks good for such case